gpumon/gpumon_cli.ml

```diff
@@ -1,26 +1,28 @@
-
 (* Gpumon CLI *)
 
-module Cmds = Gpumon_interface.RPC_API(Cmdlinergen.Gen ())
+module Cmds = Gpumon_interface.RPC_API (Cmdlinergen.Gen ())
 
 let version_str description =
-  let maj,min,mic = description.Idl.Interface.version in
+  let maj, min, mic = description.Idl.Interface.version in
   Printf.sprintf "%d.%d.%d" maj min mic
 
 let default_cmd =
-  let doc = String.concat "" [
-      "A CLI for the GPU monitoring API. This allows scripting of the gpumon daemon ";
-      "for testing and debugging. This tool is not intended to be used as an ";
-      "end user tool"] in
-  Cmdliner.Term.(ret (const (fun _ -> `Help (`Pager, None)) $ const ())),
-  Cmdliner.Term.info "gpumon_cli" ~version:(version_str Cmds.description) ~doc
+  let doc =
+    String.concat ""
+      [
+        "A CLI for the GPU monitoring API. This allows scripting of the gpumon \
```

Comment:
This is odd here, I usually prefer
```
[ "foo"
 ; "bar"
 ; "x"
 ]
```

Is there a way to avoid that extra newline at the beginning of the bracket?

Reply:
I'll see if there's an option for it. I guess the reasoning is that if an element is to be added at the beginning only a line added shows in the diff, instead of one replaced and one added.

This one looks particularly bad probably because it's building a simple string with the concat.

Reply:
This setting looks controlled by the `dock-collection-brackets` flag

these are the options it gives:
true:
```
[
  "foo"
; "bar"
; "x"
]
```
false:
```
[ "foo"
; "bar"
; "x" ]
```
So we have a sparse one that minimizes diffs and a compact one. Although it's more sparse I would lean towards the first style. Either would work fine.

Reply:
Yeah the `]` should be on a separate line yes to minimize diffs, ok lets keep the settings as you have them now.